SortingAlgorithms/cycle_sort.cpp

```diff
@@ -0,0 +1,73 @@
+#include <iostream>

```

Comment:
Try to include a comment containing the **Problem statement** with **Sample Test Cases** that have been tested with the code from the next time. 👨🏻‍💻 
Refer : [CONTRIBUTING.md](https://github.com/akashchouhan16/Data_Structures_And_Algorithms/blob/master/CONTRIBUTING.md "View Guidelines")